.github/workflows/flowzone.yml

```diff
@@ -26,5 +26,5 @@ jobs:
     with:
       custom_test_matrix: >
         {
-          os: ["ubuntu-latest", "macos-latest", "actuated-arm64-1cpu-2gb"]
+          os: ["ubuntu-latest", "macos-latest", "ARM64"]
```

Comment:
I suppose this is enough, rather than specifying `[ ..., ["self-hosted", "ARM64"]]`?

Reply:
We should be clear with the `"self-hosted"` tag IMO, it improves visibility and searchability.